src/Symfony/Component/Workflow/Attribute/Transition.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\Attribute;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+#[\Attribute(\Attribute::TARGET_CLASS_CONSTANT)]
+final class Transition
+{
+    public function __construct(
+        public string|array $froms = [],
+        public string|array $tos = [],
+        public ?string $name = null,
```

Comment:
Do we even need a name at all ? What is the use case for defining a transition name that is not equal to the constant value (making it impossible to use the constant to reference the transition) ?